main.py

```diff
@@ -634,6 +634,46 @@ def run(self):
     return rid
 
 
+@app.post("/dds/att/")
+async def set_dds_attenuation(device: str, channel: int, value: float):
+    """Sets the attenuation of the given DDS channel.
+
+    Args:
+        device: The DDS device name described in device_db.py.
+        channel: The DDS channel number. For Urukul, there are 4 channels, from 0 to 3.
+        value: The attenuation to set. For Urukul, the valid range is from 0dB to -31.5dB.
+          The value is the absolute value of the actual attenuation, e.g., 10 for -10dB.
+    """
+    if device not in configs["dds_devices"] or channel not in configs["dds_devices"][device]:
+        logger.exception("The DDS device %s CH %d is not defined in config.json.", device, channel)
+        return
```

Comment:
Since this function returns something in usual cases, please explicitly write `return None`.

See also: [PEP8](https://peps.python.org/pep-0008/#programming-recommendations)
> Be consistent in return statements. Either all return statements in a function should return an expression, or none of them should. If any return statement returns an expression, any return statements where no value is returned should explicitly state this as `return None`, and an explicit return statement should be present at the end of the function (if reachable)

Reply:
As you suggested above, I returned -1 instead of `None`.